tests/io/test_fsspec.py

```diff
@@ -586,6 +597,25 @@ def test_writing_avro_file_gcs(generated_manifest_entry_file: str, fsspec_fileio
     fsspec_fileio_gcs.delete(f"gs://warehouse/{filename}")
 
 
+@pytest.mark.gcs
+def test_fsspec_pickle_roundtrip_gcs(fsspec_fileio_gcs: FsspecFileIO) -> None:
+    _test_fsspec_pickle_round_trip(fsspec_fileio_gcs, "gs://warehouse/foo.txt")
+
+
+def _test_fsspec_pickle_round_trip(fsspec_fileio: FsspecFileIO, location: str) -> None:
+    serialized_file_io = pickle.dumps(fsspec_fileio)
```

Comment:
I just realized that we use both `fileio` and `file_io` in the codespace: (e.g. `fsspec_fileio`, `load_file_io`). I would be good if we could consistently use one of them. This may be done in a separate PR.

Reply:
Agreed! I can take up renaming in a separate PR so it's easier to review.